### PR TITLE
SOC-178 - SRP implementation

### PR DESCRIPTION
**Pull request by jborozdina**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/synereo/gloseval/pull/31" target="_blank">ReviewNinja

**Comment by henrytill** [comment]
Any reason that this is PR can't be made against `staging`?

I don't really understand what this `S1` branch is all about...

cc @n10n @edeykholt

File: build.sbt [hunk 1]
```diff
@@ -81,6 +81,8 @@ lazy val coreDeps = Seq(
   "org.json4s"                 %% "json4s-native"          % json4sVersion,
   "org.mongodb"                %% "casbah"                 % "2.6.4" exclude("org.slf4j", "slf4j-api"),
   "org.scalaj"                 %% "scalaj-http"            % "2.0.0",
+  "org.bouncycastle"            % "bcprov-jdk15on"         % "1.54",
+  "org.scalatest"               % "scalatest_2.10"         % "2.2.1" % "test",
```
Comment (henrytill) [inline build.sbt]: ScalaTest 2.2.6 should be fine.  (I'm using it in both SpecialK and AgentServices)

**Comment by edeykholt** [comment]
This change just needs to be coordinated with the UI and GLoSEval importer updates, before going into the 1.0 branch.  Otherwise, the UI and importer login will be broken.

**Comment by stephensong** [comment]
Need some guidance as to how to go about testing (and integrating) this.

**Pull request merged**